src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/KeysTable.vue

```diff
@@ -206,37 +204,37 @@ export default Vue.extend({
     };
   },
   computed: {
-    hasPermission(): boolean {
+    canImportFromToken(): boolean {
+      // Can the user import cwrtificate from hardware token
+      return this.$store.getters.hasPermission(Permissions.IMPORT_SIGN_CERT);
+    },
+
+    showRegisterCertButton(): boolean {
+      // Decide if the user can register a certificate
       return this.$store.getters.hasPermission(
-        Permissions.ACTIVATE_DEACTIVATE_TOKEN,
+        Permissions.SEND_AUTH_CERT_REG_REQ,
       );
     },
-    showRegisterCertButton(): boolean {
-      if (
-        this.hasPermission &&
-        this.$store.getters.hasPermission(Permissions.SEND_AUTH_CERT_REG_REQ)
-      ) {
-        return true;
-      }
-      return false;
+
+    canCreateAuthCsr(): boolean {
+      return this.$store.getters.hasPermission(
+        Permissions.GENERATE_AUTH_CERT_REQ,
+      );
+    },
+    canCreateSignCsr(): boolean {
+      return this.$store.getters.hasPermission(
+        Permissions.GENERATE_AUTH_CERT_REQ,
```

Comment:
Should be generate_sign_cert_req?

Reply:
Seems that these two methods are not used at all. This is checked in KeyRow component.

Reply:
I will remove these.